modules/ocf_stats/manifests/munin.pp

```diff
@@ -97,7 +98,9 @@
           sethandler   => 'fcgid-script',
           auth_require => 'all granted',
         }
-      ];
+      ],
+
```

Comment:
Personally I'd remove this extra line so that it's clearer what section the require belongs to.

Reply:
I put the extra line because we're arranging attributes by category here and I thought `require` and `directories` don't fit into one category. Thoughts?